docker/apy.Dockerfile

```diff
@@ -54,7 +54,7 @@ RUN cd /root/chromium-compact-language-detector && \
 
 WORKDIR /source
 
-ADD https://raw.githubusercontent.com/unhammer/apertium-get/master/apertium-get .
+ADD https://raw.githubusercontent.com/sushain97/apertium-get/master/apertium-get .
```

Comment:
@Androbin still seem to be some unrelated things in the diff?

Reply:
This change was on `issue-55-suggestions` and it was more recent than `master`.
So I thought I'd better not remove it.

Reply:
It should be `unhammer`. I think I just made the change in `master`? There's also more unrelated stuff in `localization.js`.

Reply:
The changes in `localization.js` include some minor fixes e.g. imports/exports which previously broke the build on the merge.

Reply:
Ah, I'm not sure I understand why it broke the build. Regardless, this should be fixed. Do you want to try completing this feature or were you just trying to de-conflict?

Reply:
I was just trying to de-conflict the existing work.
I did my best to figure out how to merge the two branches.